src/features/measurement.rs

```diff
@@ -65,9 +65,9 @@ impl Measurement<Width> for Min {
     {
         let ctrl = CfgWidthFunction::from_cfg(cfg);
         grid_widths(&records, &ctrl)
-            .map(|r| r.min().unwrap_or(0))
+            .flatten()
```

Comment:
Are you sure it is correct?

Reply:
Applies to all the .flatten()

Reply:
yep. `grid_widths` returns an iterator of iterators. `flatten` flattens the iterators into a single iterator of all values

Reply:
actually, this one is wrong but the rest are right. This particular case is taking the max of the min, rather than the global max

Reply:
actually, you're right (At first glance I was thinking all of wrong)